pages/common/q.md

```diff
@@ -0,0 +1,23 @@
+# q
+
+> Execute SQL-like queries on .csv and .tsv files.
+
+- Query simple .csv file (with header for column name mapping):
+
+`q -d, -H "SELECT * from {{path/to/file}} where answer = 42"`
```

Comment:
Also, sorry for the force push. 
Force of habit with smaller commits like this.